include/oneapi/dpl/pstl/algorithm_fwd.h

```diff
@@ -48,6 +48,14 @@ template <class _IsVector, class _ExecutionPolicy, class _RandomAccessIterator,
 bool
 __pattern_any_of(__parallel_tag<_IsVector>, _ExecutionPolicy&&, _RandomAccessIterator, _RandomAccessIterator, _Pred);
 
+template <typename _DifferenceType, typename _F, typename... _Iterators>
+auto
+__brick_walk_n(_DifferenceType, _F, /*__is_vector=*/ std::false_type, _Iterators...) noexcept;
```

Comment:
`__brick_walk_n` is unused. Is it needed?

Reply:
removed